ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/customercenter/CustomerCenterActivity.kt

```diff
@@ -0,0 +1,31 @@
+package com.revenuecat.purchases.ui.revenuecatui.customercenter
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.ui.Modifier
+
+internal class CustomerCenterActivity : ComponentActivity() {
+    companion object {
+        internal fun createIntent(context: Context): Intent {
+            return Intent(context, CustomerCenterActivity::class.java)
+        }
+    }
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        setContent {
+            CustomerCenter(
```

Comment:
Once we add the listener in the `Purchases` class, I guess we will need to reference it here right? (Probably left for future PRs)

Reply:
yes! the listener is empty atm since the `CustomerCenter` composable doesn't accept the callbacks